tests/Functional/ServiceTest.php

```diff
@@ -0,0 +1,88 @@
+<?php
+
+namespace Tests\Functional;
+
+use Basis\Nats\Client;
+use Basis\Nats\Message\Payload;
+use Basis\Nats\Service\Service;
+use Tests\FunctionalTestCase;
+use Tests\Utils\TestEndpoint;
+
+class ServiceTest extends FunctionalTestCase
+{
+    private bool $tested = false;
+
+    private function createTestService(): Service
+    {
+        /** @var Client $client */
+        $client = $this->createClient([
+            'host' => 'hermes.internal'
```

Comment:
you can configuration local runner using phpunit.xml via `NATS_HOST`\
just copy phpunit.xml.dist and replace all you need